src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/card/card.component.scss

```diff
@@ -0,0 +1,40 @@
+@use './src/styles/vendor/variables' as vv;
+@use './src/styles/defaults/mixins';
+
+$card-font-min-width: 320px;
+$card-font-max-width: 2048px;
+$card-font-min-size: 12px;
+$card-font-max-size: 21px;
+
+.card {
+  @include mixins.fluid-font-size(
+    $card-font-min-width,
+    $card-font-max-width,
+    $card-font-min-size,
+    $card-font-max-size
+  );
+  border: 0.5px solid vv.$gray-300;
+  border-radius: 3px;
+  height: 100%;
+
+  .card-body {
+    padding-top: 40px !important;
+
+    .card-title {
+      left: -0.6rem;
+      position: absolute;
+      top: -0.3rem;
+    }
+  }
+}
+
+.no-center {
+  left: unset;
+  position: unset;
+  top: unset;
+  transform: unset;
+}
+
+.content-highlight {
+  font-weight: bold;
```

Comment:
Is all of this needed? I'd defer styling to the latest stage possible. Styling is complex to maintain and complicates productization: browser upgrades may alter it, as well as Bootstrap/framework upgrades, and for downstream branding we need to do a lot of tweaking. Additionally, putting that at component CSS may complicate the downstream branding too.

I'd love to see how the cards looked with the default Bootstrap styling and then only perform the minimal set of adjustments required.